src/pages/Login.js

```diff
@@ -1,9 +1,99 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { saveUser } from '../actions';
+
+class Login extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      email: '',
+      password: '',
+      disable: true,
+    };
+
+    this.HandleChange = this.HandleChange.bind(this);
+    this.FormValidate = this.FormValidate.bind(this);
+    this.OnSubmitUser = this.OnSubmitUser.bind(this);
+  }
+
+  HandleChange({ target }) {
+    const { name, value } = target;
+    this.setState({
+      [name]: value,
+    });
+    this.FormValidate();
```

Comment:
Como é necessário confirmar que a validação será feita apenas após a atualização do estado, é importante que garanta isso. Uma sugestão é:

```suggestion
    }, () => this.FormValidate();
```